src/backend/distributed/sql/udfs/create_distributed_table_concurrently/latest.sql

```diff
@@ -0,0 +1,14 @@
+CREATE FUNCTION pg_catalog.create_distributed_table_concurrently(table_name regclass,
+                                                                 distribution_column text,
+                                                                 distribution_type citus.distribution_type DEFAULT 'hash',
+                                                                 colocate_with text DEFAULT 'default',
+                                                                 shard_count int DEFAULT NULL)
```

Comment:
create_distributed_table_concurrently doesn't block writes while the table is been created. 
Can two distributed tables be created in parallel as of now? If they aren't supported and if we plan to support them, won't that creation be concurrent? For me 'concurrent' is giving that impression.

Reply:
This follows the semantics of CREATE INDEX CONCURRENTLY and other CONCURRENTLY commands in PostgreSQL

https://www.postgresql.org/docs/current/sql-createindex.html#SQL-CREATEINDEX-CONCURRENTLY

Reply:
Thanks. Got it now.

Reply:
You do bring up a good point that create_distributed_table can operate on two separate tables concurrently, whereas create_distributed_table_concurrently currently can't. We should document this and make sure it does not cause issues.